src/v/cluster/types.h

```diff
@@ -124,6 +125,16 @@ struct prepare_group_tx_request {
 struct prepare_group_tx_reply {
     tx_errc ec;
 };
+struct commit_group_tx_request {
```

Comment:
can you describe some where what is the relationship between the request/reply types being added at the cluster:: scope level and the batches are written to the logs as result?

Reply:
I've described the relationship in the cover letter